src/plugins/scopes/ScopesPlugin.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+
+import BasePlugin from '../BasePlugin';
+import ScopesApi from './ScopesApi';
+import type Program from '../../elements/types/Program';
+import type Node from '../../elements/Node';
+
+export default class ScopesPlugin extends BasePlugin {
```

Comment:
Why? It seems as pretty basic, like i would say you should abstract something if there is something to abstract and since js doesn't have entities like interfaces or something.

Would advise you to document that somewhere in the wiki